styles/pronouns.cc/catppuccin.user.css

```diff
@@ -158,105 +158,43 @@
       --bs-navbar-brand-hover-color: @text !important;
       --bs-navbar-toggler-border-color: fade(@crust, 10%) !important;
     }
+    //@see https://github.com/catppuccin/userstyles/pull/1140#discussion_r1703105596
```

Comment:
No need to put a comment here, if you really want to though link to the GitHub section instead of the discussion comment.